### PR TITLE
Task bodies

### PR DESCRIPTION
**Pull request by athith-g**
I have created three TES tasks (and their associated tests):

- **Task 1**: Take the contents of an input file and produce an output file with an uppercase version of the contents.
- **Task 2**: Take the contents of a _Crypt4GH-encrypted_ input file and produce an output file with the decrypted contents.
- **Task 3**: Take the contents of a _Crypt4GH-encrypted_ input file and produce an output file with the uppercase version of the contents.

Essentially, I want the middleware to turn Task 1, the [uppercase task](https://github.com/elixir-cloud-aai/protes-middleware-crypt4gh/compare/task_bodies?expand=1#diff-0678a505db851106224932fb167ef35e252624edae41f83cb9cc71583c798ccfR7) (with a crypt4gh file and secret key file as additional inputs), into Task 3, the [uppercase-with-decryption task](https://github.com/elixir-cloud-aai/protes-middleware-crypt4gh/compare/task_bodies?expand=1#diff-0678a505db851106224932fb167ef35e252624edae41f83cb9cc71583c798ccfR80).

I wanted to know if there are any glaring issues with Task 3 before I start hashing out the implementation details of the middleware, since Task 3 is ultimately going to the be the output of the middleware. Is it good enough to get started on the middleware development?

### CODE REVIEW

File: .gitignore
Comment (uniqueg) [inline .gitignore:1]: Consider generating Python-focused `.gitignore` contents, e.g., via GitHub itself, or from [here](https://www.toptal.com/developers/gitignore/) (would give you [theses](https://www.toptal.com/developers/gitignore/api/python) contents). But please do so in a separate PR to keep concerns separate.

File: Dockerfile
Comment (uniqueg) [inline Dockerfile:1]: Looks fine, but as above, please open separate PRs to keep concerns separate. A PR can never be too small, only too large.

File: README.md
Comment (uniqueg) [inline README.md:1]: Perhaps add a `README.md` file with an organization-typical top-level structure (e.g., proTES) in a separate PR (leave contents of non-boilerplate sections empty, or put your best guess). Then continuously update it along with other PRs, as required.

File: tests/tasks/test_tasks.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+import requests
+
+from task_bodies import (
+    output_dir,
+    uppercase_task_body,
+    decryption_task_body,
+    uppercase_task_with_decryption_body
+)
+
+tes_url = "http://localhost:8090/ga4gh/tes/v1"
```
[the remark this answers is not in the document]
Comment (uniqueg) [reply tests/tasks/test_tasks.py:12]: I think this is fine for now...

File: tests/tasks/test_tasks.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+def get_task(task_id):
+    get_response = requests.get(
+        url=f"{tes_url}/tasks/{task_id}", headers=headers
+        )
+
+    return get_response
+
+
+def get_task_state(task_id):
```
[the remark this answers is not in the document]
Comment (uniqueg) [reply tests/tasks/test_tasks.py:33]: Can add sth like that at a later stage

**Review by uniqueg — CHANGES_REQUESTED** [review]
Very nice! Please add some PRs to build up your repo iteratively with things like `.gitignore`, `README.md`, `Dockerfile`, CI and so on. For inspiration, you can have a look at the work @JaeAeich did (is doing) for his repo (not saying you need all of that, just what seems useful to your case).

It's a bit dull work, but you can learn a lot - and it will pay dividends further on.

**Comment by athith-g** [comment]
Thank you for the suggestions! I broke up this PR into smaller ones, so I'll close this one once all of them are merged.

**Comment by uniqueg** [comment]
Great job - thanks a lot. Let's get these PRs merged over the next 1-2 days :)

**Pull request closed without merging**